client.go

```diff
@@ -312,6 +312,11 @@ type RemoteContext struct {
 	// afterwards. Unpacking is required to run an image.
 	Unpack bool
 
+	// NoPreservation is a boolean flag to specify whether to allow GC to clean
+	// layers up from the content store after successfully unpacking these
+	// contents to the snapshotter.
+	NoPreservation bool
```

Comment:
I know this naming was already discussed... and sorry to bikeshed, but  this doesn't feel quite right... so I'll suggest one more name... take it or leave it: `DiscardContent` ... or some variation on `Discard`

Reply:
Thanks for the comment. I fixed the term to `DiscardContent`. PTAL.
cc: @dmcgowan

Reply:
I'll take another look at it this week. I intend to do a deeper review of some of the related code and overall design.